panels/configspec.ini

```diff
@@ -16,6 +16,30 @@ panel_rooms = string_list(default=list("panels_1", "panels_2", "panels_3", "pane
 # deprecated "peglegs_email_signature" from the main repo.
 panels_email_signature = string(default=" - MAGFest Panels Team")
 
+# A list of social media fields collected from panelists.
+# The values in the list will be "fieldified" – converted to field names
+# (lowercased, spaces and special characters removed, CamelCase to
+# under_scored) and added as attributes on the PanelApplicant class.
+social_media = string_list(default=list("Facebook", "Twitter", "Instagram", "LinkedIn"))
+
+# These are used as templates to generate social media links, in case a
+# panelist provides a username instead of a direct link to their social media
+# page. The keys should match "fieldified" values in the "social_media" list.
```

Comment:
In almost all other cases our key names are irrelevant, so it's super good that you added these notes. :)